cmd/compose/build.go

```diff
@@ -110,16 +110,16 @@ func buildCommand(p *projectOptions, backend api.Service) *cobra.Command {
 	cmd.Flags().StringArrayVar(&opts.args, "build-arg", []string{}, "Set build-time variables for services.")
 	cmd.Flags().StringVar(&opts.ssh, "ssh", "", "Set SSH authentications used when building service images. (use 'default' for using your default SSH Agent)")
 	cmd.Flags().Bool("parallel", true, "Build images in parallel. DEPRECATED")
-	cmd.Flags().MarkHidden("parallel") //nolint:errcheck
+	cmd.Flags().MarkHidden("parallel") // nolint:errcheck
```

Comment:
note that these are not really correct for `nolint` comments; see the GoLangCI-lint documentation  https://golangci-lint.run/usage/false-positives/

> Use `//nolint` instead of `// nolint` because machine-readable comments should have no space by Go convention.



I also opened a ticket in Golang for the go1.19 update, which seems to have some bugs around this as well; https://github.com/golang/go/issues/53835

Reply:
😬 😂 .. and Go maintainers rejected my issue, and said that "the linters are not right"

Reply:
Well... I just changed it to use `//nolint` anyways

Reply:
oh; well, re-reading again; not entirely; the space shouldn't be there, but the format "must" have a colon (`:`) and not a space after (?)